org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessModalDialog.java

```diff
@@ -74,14 +74,23 @@ public ProcessModalDialog(int WindowNo, ProcessInfo pi, boolean autoStart)
 	 * @param WindowNo
 	 * @param pi
 	 */
-	public ProcessModalDialog(EventListener<Event> listener, int WindowNo, ProcessInfo pi)
+	public ProcessModalDialog(EventListener<Event> listener, int WindowNo, boolean isReRun, ProcessInfo pi)
```

Comment:
Please update javadoc and keep the existing public constructor so that it doesn't break backward compatibility (i.e add public ProcessModalDialog(EventListener<Event> listener, int WindowNo, ProcessInfo pi) {  this(listener, WindowNo, true, pi); } ). When changing public method or constructor, if it is simple to maintain existing source and binary compatibility, we would want to keep it.